internal/cli/terraform_fetcher.go

```diff
@@ -219,10 +219,22 @@ func (f *organizationResourceFetcher) FetchData(ctx context.Context) (importData
 	}
 
 	for _, org := range orgs {
+		organization := org.(*management.Organization)
 		data = append(data, importDataItem{
-			ResourceName: "auth0_organization." + sanitizeResourceName(org.(*management.Organization).GetName()),
-			ImportID:     org.(*management.Organization).GetID(),
+			ResourceName: "auth0_organization." + sanitizeResourceName(organization.GetName()),
+			ImportID:     organization.GetID(),
 		})
+
+		conns, err := f.api.Organization.Connections(ctx, organization.GetID())
```

Comment:
Same as the others, these requests are only for detecting if there is at least one enabled connection on the organization, otherwise we skip. The precise number of enabled connections does not matter.